flocker/provision/_install.py

```diff
@@ -292,9 +390,11 @@ def task_install_flocker(
             run_from_args([
                 "add-apt-repository", "-y", "ppa:james-page/docker"]),
```

Comment:
:question: Unrelated but there should be a comment here saying why this is necessary. Is it still? Will there be a release of Docker which stops it being necessary? If so, let's make an issue to get rid of it.